fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -777,9 +777,22 @@ public void testSearchQuantity_Range_missing() throws Exception {
 
     @Test
     public void testSearchQuantity_Exponent() throws Exception {
+        // Target value is 1.2E+2 (115-125)
+
         // Value 1.2E+2 should return the value
-        // The value is extracted, and stored in the values tables.
-        assertSearchReturnsSavedResource("Quantity-withExponent", "1.2E+2");
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "1.2E2");
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "1.2E+2");
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "1.2e2");
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "1.2e+2");
+
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "lt1e2");
+        assertSearchReturnsSavedResource(     "Quantity-withExponent", "gt1e2");
```

Comment:
After a discussion, this is actually checking "less than (95-100)", so this should not find it. So need to update the tests to test the correct range or update the expected result.

Reply:
its actually checking just "less than 100".  i changed the expected result and added a comment to explain.